scripts/scarbfmt.sh

```diff
@@ -1,12 +1,10 @@
 #!/usr/bin/env bash
 
-SNFOUNDRY_ROOT="$(git rev-parse --show-toplevel)"
-
-pushd "$SNFOUNDRY_ROOT" || exit
-
-find . -type f -name "Scarb.toml" -execdir sh -c '
-  echo "Running \"scarb fmt\" in directory: $PWD"
-  scarb fmt
-' \;
-
-popd || exit
+output=$(find . -type f -name "Scarb.toml" -execdir sh -c '
+    echo "Running \"scarb fmt --check\" in directory: $PWD"
+    scarb fmt --check
+' \;)
+echo "$output"
+if grep -iq "Diff" <<< "$output"; then
+    exit 1
+fi
```

Comment:
nit. bash generally uses 2 spaces for indentation
you can easily format code with `shfmt -i 2 <file>`
```suggestion
output=$(find . -type f -name "Scarb.toml" -execdir sh -c '
    echo "Running \"scarb fmt --check\" in directory: $PWD"
    scarb fmt --check
' \;)
echo "$output"
if grep -iq "Diff" <<<"$output"; then
  exit 1
fi
```